mrjob/py2.py

```diff
@@ -0,0 +1,170 @@
+"""Minimal utilities to make Python work for 2.6+ and 3.3+
+
+Strategies for making `mrjob` work across Python versions:
+
+Bytes vs. Unicode
+-----------------
+
+It's tempting to use `from __future__ import unicode_literals` and require
+that all non-byte strings be unicode. But that doesn't really make sense for
+Python 2, where str (bytes) and unicode can be used interchangeably.
+
+So really our string datatypes fall into two categories, bytes, and
+"strings", which are either ``str`` (i.e., bytes) or ``unicode`` in Python 2,
```

Comment:
To be clear, I'm only suggesting you change the terminology, not anything else. "text" is unambiguous where your usage of "strings" is unusual.